lib/openstax/aws/deployment_base.rb

```diff
@@ -2,6 +2,7 @@ module OpenStax::Aws
   class DeploymentBase
 
     attr_reader :env_name, :region, :name, :dry_run
+    mattr_reader :tags, default: {}.with_indifferent_access
```

Comment:
instead of creating 2 heap objects, you could do HashWithIndifferentAccess.new here if you want.

Reply:
:-)